integrationtests/src/test/java/io/kroxylicious/proxy/schema/validation/JsonSyntaxValidationIT.java

```diff
@@ -250,4 +252,24 @@ private static void assertInvalidRecordExceptionThrown(Future<RecordMetadata> in
                 .hasMessageContaining(message);
     }
 
+    private CreateTopicsResult createTopics(Admin admin, NewTopic... topics) {
```

Comment:
hmm, wonder why this test is creating the topics directly against the backing cluster. Does it pass if we use the tester to create the topics via the proxy?